Dockerfile

```diff
@@ -1,20 +1,7 @@
-FROM google/dart:1.24.3
+FROM google/dart:2.1.0
 
-# Expose env vars for git ssh access
-ARG GIT_SSH_KEY
```

Comment:
Why is this being removed?  Is it possible to point to git refs in `pubspec.yaml` without this now?

Reply:
Could be my mistake. I thought it was unnecessary. Is it just for when we want to use a git ref in `pubspec.yaml`?

Reply:
@corwinsheahan-wf yep... can't run a CI build pointing to a git ref without this stuff.

Reply:
No, this should be re-added